packages/core/src/helpers/chain.ts

```diff
@@ -3,16 +3,27 @@ import { DEFAULT_SUPPORTED_CHAINS, ChainId } from '../constants'
 export const getChainById = (chainId: ChainId) =>
   DEFAULT_SUPPORTED_CHAINS.find((network) => network.chainId === chainId)
 
-export const getExplorerAddressLink = (address: string, chainId: ChainId): string | '' =>
-  getChainById(chainId)?.getExplorerAddressLink(address) || ''
+export const getExplorerAddressLink = (address: string, chainId: ChainId): string | '' => {
+  console.warn('getExplorerAddressLink is deprecated, can call with Chain directly')
```

Comment:
Extract to separate method: 
`deprecationWarning('getExplorerAddressLink is deprecated, can call with Chain directly')`